polkadot/node/core/pvf/src/execute/queue.rs

```diff
@@ -58,6 +60,7 @@ slotmap::new_key_type! { struct Worker; }
 
 #[derive(Debug)]
 pub enum ToQueue {
+	UpdateActiveLeaves { update: ActiveLeavesUpdate, ancestors: Option<Vec<Hash>> },
```

Comment:
Why `Option<Vec>` can't it just be `Vec` ?

Reply:
ActiveLeavesUpdate can have no active leaf but only deactivated, so no ancestors

Reply:
what would make the most sense to me is to define a new ActiveLeavesUpdate for this purpose like this:
```
pub struct ActiveLeavesUpdate {
	/// New relay chain block of interest plus its ancestors.
	pub activated: Option<(ActivatedLeaf, Vec<Hash>)>,
	/// Relay chain block hashes no longer of interest.
	pub deactivated: SmallVec<[Hash; ACTIVE_LEAVES_SMALLVEC_CAPACITY]>,
}
```

This would be more expressive